policyengine_us/parameters/gov/contrib/congress/romney/family_security_act_two_point_o/ctc/child_cap.yaml

```
@@ -0,0 +1,10 @@
+description: Under the Family Security Act 2.0, the Child Tax Credit will be limited to this number of children per filing unit.
```

How about this kind of phrasing?
```suggestion
description: Senator Mitt Romney proposed limiting the Child Tax Credit to this number of children per filer under the Family Security Act 2.0.
```